authcrypt/authcrypt.h

```diff
@@ -23,6 +23,7 @@
 #include "mbedtls/cipher.h"
 #include "mbedtls/entropy.h"
 #include "mbedtls/ctr_drbg.h"
+#include "mbedtls/platform.h"
```

Comment:
In this case it's better to have `mbedtls_platform_context*` as forward declaration , instead of adding include for the header. I prefer minimal includes in header files, unless really needed, and keep the include in the `.cpp` file